weed/shell/command_volume_list.go

```diff
@@ -172,6 +176,11 @@ func (c *commandVolumeList) writeDiskInfo(writer io.Writer, t *master_pb.DiskInf
 		if c.isNotMatchDiskInfo(vi.ReadOnly, vi.Collection, vi.Id) {
 			continue
 		}
+		if *c.writable {
+			if c.volumeSizeLimitMb >= vi.Size {
```

Comment:
there could be other conditions to determine whether a volume is writable, e.g., being readonly, or a remote volume, ec volumes, etc.